.github/workflows/main.yml

```diff
@@ -53,7 +53,7 @@ jobs:
       - name: Web Build
         run: |
           mkdir -v -p build/web/$EXPORT_NAME
-          cd $EXPORT_NAME
+          cd build/web/$EXPORT_NAME
           godot -v --export "HTML5" ../build/web/index.html
```

Comment:
This is correct? does it need to be $EXPORT_NAME.html ?

Reply:
no, the index.html is the file GitPages looks for to start serving the web request